Containers/nextcloud/start.sh

```diff
@@ -28,18 +28,16 @@ fi
 set -x
 IPv4_ADDRESS_APACHE="$(dig nextcloud-aio-apache A +short | grep '^[0-9.]\+$' | sort | head -n1)"
 IPv6_ADDRESS_APACHE="$(dig nextcloud-aio-apache AAAA +short | grep '^[0-9a-f:]\+$' | sort | head -n1)"
```

Comment:
Unfortunately I missed that we need for apache to start in order to get its ip-address... @Zoey2936 do you have a better idea to fix this? I dont see any unfortunately...

Reply:
maybe a volume which only contains the php socket and is shared between the needed containers?

Reply:
Then we run into permission problems and things like the socket not getting correctly deleted when the process ends...

But I found a solution with a fake service